src/main/java/com/hivemq/HiveMQServer.java

```diff
@@ -19,6 +19,7 @@
 import com.codahale.metrics.MetricRegistry;
 import com.google.common.util.concurrent.ListenableFuture;
 import com.google.inject.Injector;
+import com.hivemq.configuration.service.entity.Listener;
```

Comment:
the added dependencies are probably not needed

Reply:
Done